README.md

```diff
@@ -23,6 +23,17 @@ This mod works with a single dll file but we also provide a way to downpatch the
 installing the latest Microsoft Visual C++ Redistributable version could fix it : https://aka.ms/vs/17/release/vc_redist.x64.exe.
 - The Windows console tends to freeze preventing you from sending or receiving any items. You must Alt+Tab, click on the console and press enter to refresh it.
 
+## Building Locally
+
+1. Clone the repository (`git clone https://github.com/Marechal-L/Dark-Souls-III-Archipelago-clieng.git`).
```

Comment:
Thanks for writing these! They helped me get building. Perhaps you have `submodule.recurse` and I don't cause I also had to do the `git submodule` commands to pull down those before it would build.